interface_config.js

```diff
@@ -20,6 +20,8 @@ var interfaceConfig = { // eslint-disable-line no-unused-vars
     // the toolbar buttons line is intentionally left in one line, to be able
     // to easily override values or remove them using regex
     MAIN_TOOLBAR_BUTTONS: ['microphone', 'camera', 'desktop', 'invite', 'hangup'], // jshint ignore:line
+    // list of buttons for left toolbar
+    EXTENDED_TOOLBAR_BUTTONS: ['profile', 'contacts', 'chat', 'security', 'etherpad', 'settings', 'raisehand', 'fullscreen',  'filmstrip'], // jshint ignore:line
```

Comment:
The TOOLBAR_BUTTONS already contains the list of extended toolbar buttons. That's why we have 2 lists MAIN_TOOLBAR_BUTTONS for the top toolbar and TOOLBAR_BUTTONS for the extended toolbar.